### ISSUE TITLE
[Bug]: Pipeline deploying to a different region

### ISSUE THREAD

**Issue by smc-dotexe** [aws/copilot-cli#5846]
At the moment, I have a .NET Web Api deployed using Copilot services. 
Currently a service and environment are deployed with AppRunner.

Now I'm trying to deploy a pipeline to the us-west-2 region (where the Environment and Service live).
After initializing the pipeline, and selecting either Workload or Environment. I run "copilot pipeline deploy" and go through all the steps to create it.

For some reason it deploys the pipeline in the ca-central-1 region. I do have other applications in that region but I need for it to deploy in us-west-2.

When I run "copilot app show", I can see the Environment is indeed in the us-west-2 region:

About

  Name                  flight-app
  Version               v1.33.4
  URI                   N/A
  Permissions Boundary  N/A

Environments

  Name     AccountID     Region

  staging  672368182217  **_us-west-2_**

Workloads

  Name     Type                        Environments

  api-svc  Request-Driven Web Service  staging

Pipelines

  Name

  pipeline-flight-app-staging-Pipeline-1EUB4UFXJI5F3

When I created the Environment, I made it use the credentials profile that has us-west-2.
I do have another profile that is ca-central-1 which I'm wondering if the pipeline could be confused with that somehow.

**Comment by huanjani** [comment]
Hello, @smc-dotexe!

Copilot pipelines are created in the same account and region as the **application**. Ref: https://aws.github.io/copilot-cli/docs/concepts/pipelines/#step-5-creating-your-pipeline. Is ca-central-1 where your flightapp was deployed?

**Comment by smc-dotexe** [comment]
Thank you for your response.
It seems to have been deployed to ca-central-1.
However I deleted the app to try to redeploy it in the correct region. It automatically defaults to ca-central-1. I created an AWS profile "default" and configured it to the us-west-2 region, but it still goes to the ca-central-1 region.
I must have an environment variable set elsewhere to the ca-central-1 region by default. Or I'm missing something else..

It would be ideal to have the "--region" parameter allowed for the app init or deploy.
